Heart/Message/Presentation/Request/CreateVoiceMessageRequest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Heart\Message\Presentation\Request;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateVoiceMessageRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return true;
+    }
```

Comment:
We don't need to define the `authorize()` method, since `true` is the default value.